tests/data/orbitalFramer.json

```diff
@@ -31,29 +32,61 @@
     "minheight": -1000,
     "unit": "m"
   },
-  "sensor_position": {
+  "instrument_position": {
     "unit": "km",
     "positions": [
       [1050.0, 0.0, 0.0]
     ],
     "velocities": [
       [ 0.0, 0.0, -1.0]
-    ]
+    ],
+    "ephemeris_times": [
+      50
+    ],
+		"reference_frame": 1
+  },
+  "instrument_pointing": {
+    "quaternions": [
+      [-0.707106781186547, 0.0, -0.707106781186547, 0]
+    ],
+		"angular_velocities": [
+			[0, 0, 0]
+		],
+    "ephemeris_times": [
+      50
+    ],
+    "reference_frame": 1
   },
-  "sensor_orientation": {
+  "body_rotation": {
     "quaternions": [
-      [0.0, -0.707106781186547, 0, 0.707106781186547]
-    ]
+      [1, 0, 0, 0]
+    ],
+    "ephemeris_times": [
+      50
+    ],
+    "angular_velocities": [
+      [0.0, 0.0, 0.0]
+    ],
+    "reference_frame": 1,
+    "constant_frames": [1, 2, 3],
+    "time_dependent_frames": [4, 5, 6]
   },
   "starting_detector_line": 0,
   "starting_detector_sample": 0,
+  "detector_sample_summing": 1.0,
+  "detector_line_summing": 1.0,
+  "starting_ephemeris_time": 0,
   "sun_position": {
     "positions": [
-      [100.0, 100.0, 0.0]
+      [100000.0, 100000.0, 0.0]
```

Comment:
All the other sun positions in this PR were updated from meters to km. Just want to double check that this one is correct?

Reply:
This was changed to 100,000 because the old sun position put the sun literally inside of the planet. This puts it a more realistic, but still smaller than it would be for the earth, distance away. Because the CSM API doesn't do vectors to the sun, it just does direction vectors, the magnitude doesn't impact the function output.